languages/elixir/exercises/concept/numbers/.meta/example.ex

```diff
@@ -0,0 +1,21 @@
+defmodule AssemblyLine do
+
+  @spec success_rate(integer()) :: float()
+  defp success_rate(speed) do
+    cond do
+      speed >= 9 -> 0.77
+      speed >= 5 -> 0.9
+      true       -> 1.0
+    end
+  end
```

Comment:
It's probably more natural to use `guard clauses` here as opposed to `cond`. This would also affect concepts and hints, unless the point is to avoid doing pattern matching and guards for now.
```elixir
defp success_rate(speed) when speed in 1..4, do: 1.0
defp success_rate(speed) when speed in 5..8, do: 0.9
defp success_rate(speed) when speed in 9..10, do: 0.77
```

Reply:
I have intentionally used a `cond` statement because I didn't want to introduce writing functions yet, but instead leave that for a specific concept about writing named functions, guards, and overloading functions.

I would fully expect many new people to just the cond statement inside of the first function, and since this is an unmentored exercise, it would likely then pass the tests and we don't have much control about the style used unless we implement an analyzer module for this concept.

Reply:
I could write that explicitly into the design document, if that helps to clarify things.

Reply:
I think it's definitely worth nothing when we do something a bit less idiomatic because we haven't introduced the idiomatic concepts yet. :+1:

Reply:
Agreed, will update the design doc.